src/clib/lib/enkf/enkf_main.cpp

```diff
@@ -524,13 +465,11 @@ enkf_main_case_is_initialized__(const ensemble_config_type *ensemble_config,
 }
 
 bool enkf_main_case_is_initialized(const enkf_main_type *enkf_main,
-                                   const char *case_name) {
-    enkf_fs_type *fs = enkf_main_mount_alt_fs(enkf_main, case_name, false);
+                                   const char *case_name, enkf_fs_type *fs) {
     if (fs) {
```

Comment:
This function doesn't really have anything to do with enkf_main. Its more a `fs` sort of thing, although it needs to know the ensemble size and ensemble_config. Perhaps in a future PR, change it to
```cpp
static bool
enkf_fs_is_initialized__(enkf_fs_type *fs, const ensemble_config_type *ensemble_config,
                         const int ens_size);
```
and then from enkf_main.py do:
```python
   self._fs_rotator[case].is_initializes(self.ensembleConfig(), self.getEnsembleSize())
```

Reply:
Yes, that makes a lot of sense, will create an issue 👍

Reply:
I ended up inclining `enkf_main_case_is_initialized` and removed `case_name` as it was not used. Can move the function in the same PR if you want, originally did not do it to keep the diff down, but no problem to do it.

Reply:
Also, see comment above